book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -299,6 +299,16 @@ This changes the SHA-1s of the three most recent commits in your list, so make s
 Notice that the last commit (`f7f3f6d`) in the list is unchanged.
 Despite this commit being shown in the script, because it was marked as ``pick'' and was applied prior to any rebase changes, Git leaves the commit unmodified.
 
+[[_practical_exercises_for_git_rebase ]]
+==== Practical exercises for git rebase
+
+[NOTE]
+====
+Drew DeVault made a practical hands-on guide with exercises to learn how to use `git rebase`.
+You can find it at:
+https://git-rebase.io/
```

Comment:
What about this? That way it'll be linkified.

```suggestion
You can find it at https://git-rebase.io[].
```

Reply:
Whoops, I forgot the `[]`, will go do that now. 😄